### DISCUSSION TITLE
How to change the modoboa admin and webmail (mail.) to another subdomain

### DISCUSSION THREAD

**Discussion by rodrigoslayertech** [modoboa/modoboa#2578] [Help]
How to change the modoboa admin and webmail (mail.) to another subdomain keeping the mail. to mail services?

I have changed the file conf in nginx but not effect:

server_name to port 80
from mail. to webmail.

server_name to port 443
from mail. to webmail.

Restarted nginx but no effect!

The website is still being accessed via mail. instead of webmail.

Nginx crazy

**Comment by tonioo** [comment]
You also need to update the ALLOWED_HOSTS setting and a row in your SQL database (in the django_site table or something like that).
